File: leetcode/0026-remove-duplicates-from-sorted-array/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int size = nums.size();
        if (size <= 1)
            return size;
        
        int put = 0, pick = 0;
        while (pick < size) {
            if (nums[put] != nums[pick]) {
                nums[put + 1] = nums[pick];
                ++put;
            }
            ++pick;
        }
        
        return put + 1;
    }
};

File: leetcode/0026-remove-duplicates-from-sorted-array/solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        size = len(nums)
        
        if size <= 1:
            return size
        put = 0
        pick = 0
        
        while pick < size:
            if nums[put] != nums[pick]:
                nums[put + 1] = nums[pick]
                put = put + 1
            pick = pick + 1
        return ""
        return put + 1